apps/server/src/shared/domain/entity/user.entity.ts

```diff
@@ -36,7 +36,7 @@ export class User extends BaseEntityWithTimestamps {
 	ldapDn?: string;
 
 	@Property()
-	ldapId?: string;
+	ldapId!: string;
```

Comment:
this change I need explained. Users may not have an ldapId.

Reply:
ldapId in our database is the uuid, this is the property we check to get our user from the database... we also get an uuid from iserv. We thought that we can compare these two propertys... can't we ?

Reply:
using the ldapid as uuid to identify the users is correct. However only users that actually come from an ldap actually have such an id. There also are users that are not coming from an ldap, but were created in the schoolcloud itself. The "!" would mean that you are ensuring every user has this id, which is not the case.

Reply:
removed the "!", because you're right!